src/Compiler/Checking/infos.fsi

```diff
@@ -178,15 +178,17 @@ type ILTypeInfo =
 
     member TypeInstOfRawMetadata: TypeInst
 
+[<NoComparison; NoEquality>]
+type ILMethodsType =
```

Comment:
I guess `ILMethType` is the natural name of this given the other names.

Reply:
Thanks, ILMethParentTypeInfo  is good